src/amo/components/AddonMeta.js

```diff
@@ -1,37 +0,0 @@
-import React, { PropTypes } from 'react';
```

Comment:
Okay, I'll leave that part in and let you touch it up. I think it might look a bit weird having that blue bar with only one thing in it. I'll move the install switch into the bar and then @pwalm can give us some feedback.